.github/workflows/helm-lint.yml

```diff
@@ -20,89 +20,86 @@
 name:  SDE Lint and Test Charts
 
 on:
-    push:
-      branches:
-       - main
-      paths:
-       - 'charts/**'
-
+  push:
+    branches:
+      - main
+    paths:
+      - 'charts/**'
     pull_request:
```

Comment:
needs on step further under the `push:` level so it will trigger when pushing on pull-requests

you can also add some IDE addons for validation your workflow

Reply:
Thanks for the help. corrected the indentation.